SUPPORT.md

```diff
@@ -0,0 +1,32 @@
+# Support
+
+## Issues
+
+This project uses GitHub Issues to track bugs. Please search the existing issues before filing new issues to avoid duplicates.  For new issues, file your bug as a new issue.
```

Comment:
```suggestion
This project uses GitHub Issues to track bugs. Please search the existing issues before filing new issues to avoid duplicates. For new issues, file your bug as a new issue.
```

Reply:
I wonder about saying that we use GitHub issues to track bugs. For instance, this is not true for the [Product repo](https://github.com/meilisearch/product). As this `SUPPORT.md` file will be added to all our repos, I would go for something less precise.

Reply:
What about: "This project uses GitHub Issues to track tasks and/or bugs"?